library/src/scripts/headers/ActionBar.story.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license http://www.opensource.org/licenses/gpl-2.0.php GNU GPL v2
+ */
+
+import { StoryHeading } from "@library/storybook/StoryHeading";
+import { storiesOf } from "@storybook/react";
+import { StoryContent } from "@library/storybook/StoryContent";
+
+import React from "react";
+import { ActionBar } from "@library/headers/ActionBar";
+import { MemoryRouter } from "react-router";
+import DropDown, { FlyoutType } from "@library/flyouts/DropDown";
+import DropDownItem from "@library/flyouts/items/DropDownItem";
+import DropDownItemButton from "@library/flyouts/items/DropDownItemButton";
+
+const story = storiesOf("Headers", module);
+
+story.add("ActionBar", () => {
+    return (
+        <MemoryRouter>
+            <StoryContent>
+                <StoryHeading> Default </StoryHeading>
+                <ActionBar />
+                <StoryHeading> Full </StoryHeading>
```

Comment:
I am not sure how critical it is but I would prefer to trim spaces.
```suggestion
                <StoryHeading>Full</StoryHeading>
```